file_formats/ie_formats/itm_v1.1.htm

```diff
@@ -181,18 +181,18 @@
           </tr>
           <tr>
             <td>0x0050</td>
-            <td>4 (strref)</td>
```

Comment:
strref is correct, but they're both the same size, so I don't think anyone will mind.
https://gibberlings3.github.io/iesdp/file_formats/general.htm

Reply:
There seems to be quite a bit of discrepancy as to how _strref_ is used in the documentation.  Sometimes the size of the field is given as a dword and the fact that it is a strref is added in parenthesis:
From ITM V1:

0x0050 | 4 (dword) | Unidentified Description (strref)
-- | -- | --

Other times the field is labeled as a strref directly:

0x0008 | 4 (strref) | Unidentified Name (strref)
-- | -- | --
In this case, the second "(strref)" seems a bit redundant...

Specify the preferred syntax and I'll make the ITM versions consistent with that.

Reply:
I'm not the maintainer by choice, so I don't have strong feelings about trivialities like this (hence my first comment).